### PR TITLE
spoiler and custom_spoiler

### PR DESCRIPTION
**Pull request by mkody**
`custom_spoiler` is a bit weird to use, but it's there (and I needed it, heh).
Also, `spoiler` is probably a good addition.

### CODE REVIEW

File: basc_py4chan/thread.py [hunk 1]
```diff
@@ -60,6 +61,10 @@ def imagelimit(self):
     def bumplimit(self):
         return self.topic._data.get('bumplimit') == 1
 
+    @property
+    def custom_spoiler(self):
+        return self.topic._data.get('custom_spoiler')
```
Comment (DanielOaks) [inline basc_py4chan/thread.py]: Hmm, since we're already using `.get` here, perhaps we should `.get('custom_spoiler', 0)`?

**Comment by DanielOaks** [comment]
Looks good, thanks for the contribution! Left a small comment on there

File: basc_py4chan/thread.py (hunk 1 above)
Comment (mkody) [inline basc_py4chan/thread.py]: Yup, added!

**Comment by DanielOaks** [comment]
Awesome, thanks very much for the PR. I'll make a release with this in the next few days

**Pull request merged**